tests/phpunit/tests/canonical/paged.php

```diff
@@ -26,4 +26,24 @@ public function test_redirect_canonical_with_nextpage_pagination() {
 		// Non-existing page should redirect to the permalink.
 		$this->assertCanonical( $link . '4/', $link );
 	}
+
+	public function test_redirect_missing_front_page_pagination_canonical() {
```

Comment:
Add `@ticket` number

Reply:
@mukeshpanchal27 Added the ticket number.